server/src/service-layer/request-models/AdminRequests.ts

```diff
@@ -2,12 +2,12 @@ import { Timestamp } from "firebase-admin/firestore"
 
 export interface MakeDatesAvailableRequestBody {
   /**
-   * Firestore timestamp, should represent a UTC date that is set to exactly midnight
+   * Firestore timestamp, ideally with the time information removed (set to midnight)
    */
   startDate: Timestamp
 
   /**
-   * Firestore timestamp, should represent a UTC date that is set to exactly midnight
```

Comment:
Dont think these changes are part of PR?

Reply:
Oh oops wtf my git is being weird again

Reply:
Just made a push to address the weird changes